app/processing/route-manual-ledger-to-request-editor.js

```diff
@@ -0,0 +1,23 @@
+const db = require('../data')
+const { getHoldCategoryId } = require('../holds')
+const sendManualLedgerMessage = require('../messaging/send-manual-ledger-message')
+const holdAndReschedule = require('../reschedule')
+const util = require('util')
+
+const routeManualLedgerToRequestEditor = async (paymentRequest) => {
+  const transaction = await db.sequelize.transaction()
+  try {
+    const manualLedgerMessage = { paymentRequest: paymentRequest.deltaPaymentRequest, paymentRequests: paymentRequest.completedPaymentRequests }
+    await sendManualLedgerMessage(manualLedgerMessage)
+    console.log('Payment request routed to request editor:', util.inspect(manualLedgerMessage, false, null, true))
+    const holdCategoryId = await getHoldCategoryId(paymentRequest.deltaPaymentRequest.schemeId, 'Manual ledger hold', transaction)
```

Comment:
Consider pulling `paymentRequest.deltaPaymentRequest` as a variable above and using throughout